<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qpid::DTMFPlate</class>
 <widget class="QWidget" name="Qpid::DTMFPlate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voice Easy</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget QPushButton {
	background: #ffffff;
}

#widget QPushButton:hover {
	background: #c8e6fa;
}

#widget #pushButtonDTMF0 {
	border-image: url(:/images/ui/dtmf_0.png);
}
#widget #pushButtonDTMF1 {
	border-image: url(:/images/ui/dtmf_1.png);
}
#widget #pushButtonDTMF2 {
	border-image: url(:/images/ui/dtmf_2.png);
}
#widget #pushButtonDTMF3 {
	border-image: url(:/images/ui/dtmf_3.png);
}
#widget #pushButtonDTMF4 {
	border-image: url(:/images/ui/dtmf_4.png);
}
#widget #pushButtonDTMF5 {
	border-image: url(:/images/ui/dtmf_5.png);
}
#widget #pushButtonDTMF6 {
	border-image: url(:/images/ui/dtmf_6.png);
}
#widget #pushButtonDTMF7 {
	border-image: url(:/images/ui/dtmf_7.png);
}
#widget #pushButtonDTMF8 {
	border-image: url(:/images/ui/dtmf_8.png);
}
#widget #pushButtonDTMF9 {
	border-image: url(:/images/ui/dtmf_9.png);
}
#widget #pushButtonDTMFAst {
	border-image: url(:/images/ui/dtmf_asterisk.png);
}
#widget #pushButtonDTMFPound {
	border-image: url(:/images/ui/dtmf_pound.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1" columnstretch="1,1,1">
      <property name="margin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonDTMF1">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonDTMF2">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonDTMF3">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonDTMF4">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonDTMF5">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonDTMF6">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButtonDTMF7">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButtonDTMF8">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonDTMF9">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButtonDTMFAst">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButtonDTMF0">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButtonDTMFPound">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="phone.qrc"/>
 </resources>
 <connections/>
</ui>
